app/integration_test/app_test.dart

```diff
@@ -70,6 +70,51 @@ void main() {
         find.byKey(const Key('dashboard-appbar-title-E2E')),
         findsOneWidget,
       );
+
+      // At the moment, we can't log out properly / use the navigation when
+      // signing in again. This blocks to write more integration tests. As a
+      // workaround, we put all integration test into one test.
+      //
+      // We can remove this workaround, when the following issue are resolved:
+      // * https://github.com/SharezoneApp/sharezone-app/issues/497
+      // * https://github.com/SharezoneApp/sharezone-app/issues/117
+
+      print("Test: User should be able to load groups");
```

Comment:
`log()` prints an empty message when our integration test. How can we enable the `log()`. Used the `print()` as an alternative for now.